pkg/tfvars/libvirt/libvirt.go

```diff
@@ -22,34 +26,44 @@ type Network struct {
 	IPRange string `json:"libvirt_ip_range,omitempty"`
 }
 
-// TFVars fills in computed Terraform variables.
-func (l *Libvirt) TFVars(masterCount int) error {
-	_, network, err := net.ParseCIDR(l.Network.IPRange)
+// TFVars generates libvirt-specific Terraform variables.
+func TFVars(config []byte, image string, vpcCIDR string, bridge string, masterCount int) (*Libvirt, error) {
```

Comment:
should consume type not config bytes.
vpcCidr is nt suitable in libvirt.

Reply:
> should consume type not config bytes.

Consumes structured data in 2b2e8c149.

> vpcCidr is nt suitable in libvirt.

This was addressed by #983.